### PR TITLE
Add ARM64 Support for UWP

### PR DESCRIPTION
**Pull request by RehanSaeed**
Fixes https://github.com/microsoft/AdaptiveCards/issues/2930

### CODE REVIEW

File: source/uwp/AdaptiveCards.sln [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26403.7
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28803.452
```
Comment (paulcam206) [inline source/uwp/AdaptiveCards.sln]: We can't update to VS 16 yet -- our CI hosts don't have it available yet...

File: source/uwp/Build/SignConfig.xml [hunk 2]
```diff
@@ -15,5 +15,7 @@
     <file src="__RELBINPATH__\..\..\..\s\tosign\AdaptiveCardRenderer\x64\AdaptiveCards.Rendering.Uwp.dll" signType="Authenticode" dest="__RELBINPATH__\..\..\..\s\signed\AdaptiveCardRenderer\x64\AdaptiveCards.Rendering.Uwp.dll" />
     <!-- ARM -->
     <file src="__RELBINPATH__\..\..\..\s\tosign\AdaptiveCardRenderer\ARM\AdaptiveCards.Rendering.Uwp.dll" signType="Authenticode" dest="__RELBINPATH__\..\..\..\s\signed\AdaptiveCardRenderer\ARM\AdaptiveCards.Rendering.Uwp.dll" />
+	<!-- ARM64 -->
```
Comment (paulcam206) [inline source/uwp/Build/SignConfig.xml]: oops! looks like a tab slipped in here :)

File: source/uwp/AdaptiveCards.sln (hunk 1 above)
Comment (RehanSaeed) [reply source/uwp/AdaptiveCards.sln]: Rolled back to VS 15.

File: source/uwp/Build/SignConfig.xml (hunk 2 above)
Comment (RehanSaeed) [reply source/uwp/Build/SignConfig.xml]: Fixed.

**Comment by RehanSaeed** [comment]
Hi @paulcam206, fixed the two issues you highlighted. Some of the builds also don't seem to start.

**Comment by paulcam206** [comment]
build fails with

```
##[error]C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\Microsoft\VisualStudio\v15.0\AppxPackage\Microsoft.AppXPackage.Targets(6079,5): Error : In order to compile your application for the ARM64 configuration, you must update your project's Minimum Version to the Windows 10 Fall Creators Update (Build 16299) or higher.
```

**Comment by RehanSaeed** [comment]
I've upgraded all the projects to Windows 10 Fall Creators Update (Build 16299). Over to you @paulcam206.

**Pull request merged**